lib/advSearchOk.ml

```diff
@@ -453,9 +453,7 @@ let searching_fields conf base =
           match p_getenv conf.env k with
           | Some v ->
               loop
-                (if acc = "" then v
-                else if v = "" then acc
-                else acc ^ " / " ^ v)
+                (if acc = "" then v else if v = "" then acc else acc ^ " / " ^ v)
```

Comment:
not sure but this looks like it should be a `String.concat`

Reply:
possibly!